debugging.rst

```diff
@@ -0,0 +1,84 @@
+.. _debugging:
+
+==============
+Debugging Jobs 
+==============
+
+Flux supports parallel debuggers such as Rogue Wave Software (RWS)'s
+TotalView parallel debugger (https://www.roguewave.com).
+
+----------------------------------
+Parallel Debugging using TotalView
+----------------------------------
+
+Debugging your MPI job from the beginning of
+parallel program execution:
+
+.. code-block:: sh
+
+  totalview --args flux mini run -N 2 -n 2 ./mpi-program
+
+Attaching to an already running job:
+
+.. code-block:: sh
+
+  totalview --args flux mini attach <JOBID>
+
+Note that you should use ``totalview --args flux mini attach`` whether your
+job has been launched via ``flux mini run`` or ``flux mini submit``.
+In other word, Flux currently does not allow TotalView
+to attach to your running code if you just attach it
+to the ``flux mini run`` process that controls your code.
+
+Flux provides ``MPIR_partial_attach_ok`` (https://www.mpi-forum.org/docs/mpir-specification-10-11-2010.pdf)
+so you can attach TotalView to a subset of parallel processes as well, which
+can be handy when you debug a large-scale job. Please refer to
+TotalView user guide on its subset-attach capability: https://docs.roguewave.com/en/totalview/current/html/.
```

Comment:
Since the TotalView user guide is referenced multiple times, you can created a named link rather than an anonymous one (I **think** this name is valid anywhere in this file, maybe even across the entire repo).

```suggestion
`TotalView user guide`_ on its subset-attach capability.

.. _TotalView user guide: https://docs.roguewave.com/en/totalview/current/html/
```

Reply:
Good idea. Just to make sure: here, the external hyperlink is given with the "normal" syntax?

Reply:
The Sphinx documentation describes it better than I can: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links

There is basically two ways to do it: with an inline/anonymous link or with a "ref" and "target definition". For this particular case, I recommend trying the latter since this link it used multiple times. You can define the target once, and then reference it multiple times.